### DISCUSSION TITLE
How can I style the "TabbedBar" element

### DISCUSSION THREAD

**Discussion by caspahouzer** [appcelerator/titanium_mobile#12589] [Q&A] [answered]
How it looks at the moment, you can see here
https://drive.google.com/file/d/19xyGt6w6akaAV4cko0BekpNFTkh-Nlyj/view?usp=sharing
The inactive tab is not visible. I think, the font color is white. But I have no clue how to change

```
<item name="tabSelectedTextColor">#f00</item> works
<item name="tabTextColor">#ff0</item> has no effect
```

https://github.com/appcelerator/titanium_mobile/discussions/12568 brings no new idea

**Comment by m1ga** [comment, marked as answer]
![Screenshot_20210316-185350](https://user-images.githubusercontent.com/4334997/111356824-07891d00-8689-11eb-99b6-7cbc23e43661.png)

```
    <style name="Base.Widget.Design.TabLayout" parent="Widget.Design.TabLayout">
         <item name="tabSelectedTextColor">#f00</item>
         <item name="tabTextColor">#ff0</item>
         <item name="tabTextAppearance">@style/MyCustomTabTextAppearance</item>
    </style>

    <style name="MyCustomTabTextAppearance" parent="TextAppearance.Design.Tab">
        <item name="textAllCaps">false</item>
        <item name="android:textAllCaps">false</item>
    </style>
```

Think the parent is not correct since it will remove parts like the underline and the ripple effect. So you have find the proper parent or add those again.
Got the parts from here: https://chromium.googlesource.com/android_tools/+/febed84a3a3cb7c2cb80d580d79c31e22e9643a5/sdk/extras/android/support/design/res/values/styles.xml#45
